lib/api_hammer/sinatra/halt.rb

```diff
@@ -5,7 +5,7 @@ module Sinatra
     module Halt
       # halt and render the given body 
       def halt(status, body, render_options = {})
-        throw :halt, format_response(status, body, headers)
+        throw :halt, format_response(status, body)
```

Comment:
is this a recent change? or it was not working ? there was no tests failing for this mistake ?